### PR TITLE
:bug: Fix whole_doc_chunker missing from client list

### PR DESCRIPTION
**Pull request by gkumbhat**
closes: #228 

### Changes

- Add `whole_doc_chunker` client as default client for chunkers

### CODE REVIEW

File: src/clients.rs [hunk 1]
```diff
@@ -41,7 +41,6 @@ pub mod http;
 pub use http::HttpClient;
 
 pub mod chunker;
-pub use chunker::ChunkerClient;
```
Comment (gkumbhat) [inline src/clients.rs:44]: Since this is unused, clippy was complaining

**Review by mdevino — APPROVED** [review]
Looks good to me (though I'm not super familiar with the chunking piece yet).
I've tested changes locally and verified that the whole_doc_chunk works for the two endpoints below:

* /api/v1/task/classification-with-text-generation
* /api/v1/task/server-streaming-classification-with-text-generation

**Review by declark1 — APPROVED** [review]
LGTM,

Just a couple of nit suggestions:
- I'd rename `chunker::DEFAULT_MODEL_ID` to `DEFAULT_CHUNKER_ID` instead of import aliases
- I'd move `tokenize_whole_doc` and `tokenize_whole_doc_stream` out of `chunker.rs` to `unary.rs` and `streaming.rs`, respectively, as this functionality is no longer part of the client.

**Comment by gkumbhat** [comment]
Thanks Dan. I renamed the chunker model id to `DEFAULT_CHUNKER_ID`.. For moving the functions out, I am not sure yet, since while they are not part of the client, they are providing similar functionality that this client provide 🤔  

We can discuss and address that later.

**Pull request merged**